pkg/antctl/raw/supportbundle/command.go

```diff
@@ -0,0 +1,484 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package supportbundle
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/cheggaaa/pb/v3"
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+	"golang.org/x/time/rate"
+	"gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog"
+
+	agentapiserver "github.com/vmware-tanzu/antrea/pkg/agent/apiserver"
+	"github.com/vmware-tanzu/antrea/pkg/agent/controller/noderoute"
+	"github.com/vmware-tanzu/antrea/pkg/antctl/runtime"
+	systemv1beta1 "github.com/vmware-tanzu/antrea/pkg/apis/system/v1beta1"
+	controllerapiserver "github.com/vmware-tanzu/antrea/pkg/apiserver"
+	antrea "github.com/vmware-tanzu/antrea/pkg/client/clientset/versioned"
+)
+
+const (
+	barTmpl      pb.ProgressBarTemplate = `{{string . "prefix"}}{{bar . }} {{percent . }} {{rtime . "ETA %s"}}` // Example: 'Prefix[-->______] 20%'
+	requestRate                         = 50
+	requestBurst                        = 100
+)
+
+// Command is the bundle command implementation.
+var Command *cobra.Command
+
+var option = &struct {
+	dir           string
+	labelSelector string
+}{}
+
+var remoteControllerLongDescription = strings.TrimSpace(`
+	Generate support bundles of the cluster, includes: antrea agent, antrea controller and general information of the cluster.
+	The controller support bundle will always be collected no matter whether the master Node is selected or not.
+`)
+
+var remoteControllerExample = strings.Trim(`
+  Generate support bundles of controller and agent on all Nodes and save them to current working dir
+  $ antctl supportbundle
+  Generate support bundles of specific Nodes that filtered by wildcard
+  $ antctl supportbundles *worker*
```

Comment:
It is a bit of an edge case, but if there are some files that match the wildcard expression in the current directory, the command will fail with a "cryptic" error:
```
abas-a01:antrea abas$ ./bin/antctl-darwin supportbundle antrea*
Error: accepts at most 1 arg(s), received 15
```

So for the example, I think it would make sense to use single quotes:
```
antctl supportbundles '*worker*'
```

for the help message, I would suggest: "  Generate support bundles of specific Nodes filtered by name, with support for wildcard expressions"

same thing below

Reply:
Make sense, updated.